### DISCUSSION TITLE
Documentation Improvement

### DISCUSSION THREAD

**Discussion by itizarsa** [jobrunr/jobrunr#1017] [General]
Once again, Jobrunr is awesome!

We could document more things since I have been setting it up for a new project. I'm ready to put in some time to document those, where should I start to contribute to the project?

**Comment by rdehuyss** [comment]
Hi @itizarsa, well the whole website is also open-source and we accept PR's for it.

You can find it here (you will need to install [gohugo](https://gohugo.io/)): https://github.com/jobrunr/website
